eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshGrpcBootstrap.java

```diff
@@ -30,10 +30,12 @@ public class EventMeshGrpcBootstrap implements EventMeshBootstrap {
 
     private final Registry registry;
 
-    public EventMeshGrpcBootstrap(ConfigurationWrapper configurationWrapper, Registry registry) {
+    public EventMeshGrpcBootstrap(Registry registry) {
         this.registry = registry;
-        this.eventMeshGrpcConfiguration = new EventMeshGrpcConfiguration(configurationWrapper);
-        eventMeshGrpcConfiguration.init();
+
+        ConfigService configService = ConfigService.getInstance();
```

Comment:
@Test
    public void testEventMeshTcpBootstrap() {
        ConfigService configService = ConfigService.getInstance();
        EventMeshTCPConfiguration o = configService.buildConfigInstance(EventMeshTCPConfiguration.class);
        System.out.println(o);
    }

output:
EventMeshTCPConfiguration(eventMeshTcpServerPort=10000, eventMeshTcpIdleAllSeconds=60, eventMeshTcpIdleWriteSeconds=60, eventMeshTcpIdleReadSeconds=60, eventMeshTcpMsgReqnumPerSecond=15000, eventMeshTcpClientMaxNum=10000, eventMeshTcpGlobalScheduler=5, eventMeshTcpTaskHandleExecutorPoolSize=12, eventMeshTcpMsgDownStreamExecutorPoolSize=12, eventMeshTcpSessionExpiredInMills=60000, eventMeshTcpSessionUpstreamBufferSize=100, eventMeshTcpMsgAsyncRetryTimes=3, eventMeshTcpMsgSyncRetryTimes=1, eventMeshTcpMsgRetrySyncDelayInMills=500, eventMeshTcpMsgRetryAsyncDelayInMills=500, eventMeshTcpMsgRetryQueueSize=10000, eventMeshTcpRebalanceIntervalInMills=30000, eventMeshServerAdminPort=10106, eventMeshTcpSendBackEnabled=true, eventMeshTcpSendBackMaxTimes=3, eventMeshTcpPushFailIsolateTimeInMills=30000, gracefulShutdownSleepIntervalInMills=1000, sleepIntervalInRebalanceRedirectMills=200, eventMeshEventSize=1000, eventMeshEventBatchSize=10, gtc=EventMeshTCPConfiguration.TrafficShapingConfig(writeLimit=0, readLimit=10000, checkInterval=1000, maxTime=2000), ctc=EventMeshTCPConfiguration.TrafficShapingConfig(writeLimit=0, readLimit=2000, checkInterval=1000, maxTime=10000))
return all default value,but no load from properties file.

Reply:
In EventMeshStartup, we will set the path to the root configuration file and eventually resolve to EventMeshTCPConfiguration, UT can refer to EventMeshServerTest.
This design does not take into account the scenario of creating EventMeshTCPConfiguration directly in the main process, but for use in UT you can refer to EventMeshTCPConfigurationTest.
I will follow up with optimizations that have been documented.